sechub-cli/src/mercedes-benz.com/sechub/cli/job.go

```diff
@@ -53,24 +53,44 @@ func waitForSecHubJobDone(context *Context) (status jobStatusResult) {
 			break
 		}
 
-		// PROGRESS bar ... 50 chars with dot, then next line...
-		if newLine {
-			sechubUtil.PrintIfNotSilent(strings.Repeat(" ", 29), context.config.quiet)
-			newLine = false
+		if !context.config.quiet {
+			// Progress dots
+			cursor = printProgressDot(cursor)
 		}
-		sechubUtil.PrintIfNotSilent(".", context.config.quiet)
-		if cursor++; cursor == 50 {
-			sechubUtil.PrintIfNotSilent("\n", context.config.quiet)
-			cursor = 0
-			newLine = true
-		}
-		time.Sleep(time.Duration(context.config.waitNanoseconds))
+
+		time.Sleep(time.Duration(waitNanoseconds))
+		waitNanoseconds = computeNextWaitInterval(waitNanoseconds, context.config.waitNanoseconds)
 	}
 	sechubUtil.PrintIfNotSilent("\n", context.config.quiet)
 
 	return status
 }
 
+// Print progress dots ... 50 dots, then next line...
+func printProgressDot(cursor int) int {
+	if cursor == 0 {
+		// initial identation
+		fmt.Print(strings.Repeat(" ", 29))
+	}
+
+	fmt.Print(".")
+
+	if cursor++; cursor == 50 {
+		fmt.Print("\n")
+		cursor = 0
+	}
+
+	return cursor
+}
+
+func computeNextWaitInterval(current int64, max int64) int64 {
```

Comment:
Why not using `uint64`? The values should never be negative.

Reply:
`time.Duration()` expects `int64` and not `uint64`. We avoid conversions when using `int64`.